docker/.env

```diff
@@ -17,6 +17,7 @@
 
 
 COMPOSE_PROJECT_NAME=superset
+DEV_MODE=true
```

Comment:
Is there anything differently that needs to be done when building official docker images during releases to remove this flag?